packages/apm/src/integrations/tracing.ts

```diff
@@ -202,16 +202,23 @@ export class Tracing implements Integration {
         return event;
       }
 
-      if (
-        Tracing.options.maxTransactionDuration !== 0 &&
-        Tracing._isEnabled() &&
-        event.type === 'transaction' &&
-        event.timestamp &&
-        event.start_timestamp &&
-        (event.timestamp - event.start_timestamp > Tracing.options.maxTransactionDuration ||
-          event.timestamp - event.start_timestamp < 0)
-      ) {
-        return null;
+      if (Tracing._isEnabled()) {
+        if (
+          Tracing.options.maxTransactionDuration !== 0 &&
+          event.type === 'transaction' &&
+          event.timestamp &&
+          event.start_timestamp &&
+          (event.timestamp - event.start_timestamp > Tracing.options.maxTransactionDuration ||
```

Comment:
I'd extract this to something more descriptive like `isOutdatedTransaction` or something, as its easier to understand what these 2 timing checks do